src/js/components/Select/index.d.ts

```diff
@@ -8,6 +8,18 @@ import {
   PlaceHolderType,
 } from '../../utils';
 
+export type OptionType =
```

Comment:
I believe there is still some missing from here
https://v2.grommet.io/select#options